docs/source/getting_started.rst

```diff
@@ -13,6 +13,12 @@ you can install mypy with:
 
     $ python3 -m pip install mypy
 
+Optionally you can install extra dependencies for daemon server with:
```

Comment:
Please remove this since `dmypy` is otherwise undocumented (and unsupported). This will only confuse users.